### ISSUE TITLE
Add priority feature for PRs

### ISSUE THREAD

**Issue by amanbairagi30** [amanbairagi30/merged-n-share#9]
Recently , it was suggested to add the feature of the priority of the PRs so it will be helpful  for user as they will be able to customize the order of the PR in the profile page 

This feature can be helpful if the user want to showcase his selective PRs first (based on the priority criteria) at profile page

We may take 3 priority levels as of now (open to discussion)
1. High
2. Medium
3. Low(default)

by default the priority must be set to low while the process of saving PRs in DB and user should be able to  edit the priority in my pr page

we may added the three dots to open a dialog box and then have the options to edit the priority

![image](https://github.com/user-attachments/assets/077c1337-4ce6-4c24-9854-58bd1db16a21)

**Comment by amanbairagi30** [comment]
hi @harsimran-d , would you like to give it a try ?
Open for discussion on this

**Comment by harsi-dhaliwal** [comment]
hi aman 
i think we can implement something like a pinned repos on the Github profile page 

give user ability to do six top pins and store them in a array 

the order of the array will be how they will be shown 

at most six is good i think

**Comment by harsi-dhaliwal** [comment]
the sort order of the pins can be last in on top 
like insert on the start of array 
so what ever the user pins at last will be shown first

**Comment by harsi-dhaliwal** [comment]
it will be a clean UI and a familiar UX as its same on github too

**Comment by amanbairagi30** [comment]
> hi aman i think we can implement something like a pinned repos on the Github profile page
> 
> give user ability to do six top pins and store them in a array
> 
> the order of the array will be how they will be shown
> 
> at most six is good i think

yeah , this makes sense too , yeah we can go ahead with this , also what do you think about giving the tags to the PRs which helps the user to signifies the PR more , max 2-3 tags

**Comment by harsi-dhaliwal** [comment]
for significance i think pins are there 

for tags we can add like wether it was UI, Backend, Web3 etc 

also  i think we can fetch the languages or the top languge for the repo or the pr and just show that on the card
this should be automatic form the github apis i think

**Comment by amanbairagi30** [comment]
> for significance i think pins are there
> 
> for tags we can add like wether it was UI, Backend, Web3 etc
> 
> also i think we can fetch the languages or the top languge for the repo or the pr and just show that on the card this should be automatic form the github apis i think

yeah , makes sense for now . Let me know , if you want to pick this up

**Comment by MdSadiqMd** [comment]
hey, i like to contribute to this issue
is it still available

**Comment by amanbairagi30** [comment]
hey @MdSadiqMd , it is open to contribute , feel free to make a PR for the same

**Comment by MdSadiqMd** [comment]
Please add seeders
